database/serviceStressTesting.yml

```diff
@@ -0,0 +1,39 @@
+config:
+  target: "http://localhost:4001"
+  phases:
+    - duration: 60
+      arrivalRate: 5
+      name: Warm up
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 600
+      arrivalRate: 50
```

Comment:
same comment: doesn't this mean that it creates only 50 RPS for six minutes? is this your final test?